workflows/ascc.nf

```diff
@@ -63,29 +73,44 @@ include { CUSTOM_DUMPSOFTWAREVERSIONS                   } from '../modules/nf-co
 workflow ASCC {
 
     main:
-    ch_versions = Channel.empty()
-    ch_out_merge = Channel.empty()
+    ch_versions     = Channel.empty()
+    ch_out_merge    = Channel.empty()
 
-    workflow_steps = params.steps.split(",")
+    include_workflow_steps  = params.include ? params.include.split(",") : ""
+    exclude_workflow_steps  = params.exclude ? params.exclude.split(",") : ""
```

Comment:
It would be good to have a check here to see if all elements of the list of `steps` given by the user are valid names and exit if there is an unrecognised `step` name.
I guess the valid options should be things from this list: `kmers, tiara, coverage, nt_blast, nr_diamond, uniprot_diamond, kraken, fcs-gx, fcs-adaptor, vecscreen, btk_busco, pacbio_barcodes, organellar_blast, autofilter_assembly`
Old ASCC also had `create_btk_dataset` as a step but perhaps that's not needed and it's OK to just generate a BTK dataset at the end of every run

Reply:
Now added